src/styles/bootstrap.scss

```diff
@@ -171,15 +172,19 @@
     /* stylelint-disable-next-line scss/at-extend-no-missing-placeholder */
     @extend .form-control;
 
-    border: .125rem solid $os-default-answer-border-color;
+    border: .125rem solid $os-default-gray;
     border-radius: .3125rem;
-    box-shadow: 0 .125rem .125rem $os-default-box-shadow;
+    box-shadow: 0 .125rem .125rem $os-default-gray;        
```

Comment:
The border and box-shadow values should mirror the values I've added via my PR. If I merge after you, then it should be no issue. This is what I have:

`
border: .063rem solid #757575;
border-radius: .3125rem;
box-shadow: 0 .125rem .25rem 0 hsla(0deg 0% 46% / 25%);
`

Reply:
Ahh that makes sense! For some reason, the Figma file wasn't showing me the box shadow and border numbers! I went back and apparently, I had to click on the input box a bunch of times. Did figma give you the hsla value?

Reply:
Wait I think I see it now, Did you select the HSL from the dropdown when looking at the properties?

Reply:
Getting the box shadow down was weird because the values provided by Figma were not translating to CSS. I guess in Figma they select the color they want and then can add a second color option that lightens or darkens the primary color. So I got the default color and looked at how much Maddy wanted the color adjusted. I think it was 25 percent, so with those values I came up with the next color for the box-shadow. 

Yeah, some of the values aren't available unless you click through the nested elements. Even then they are typically in pixels so have to adjust things to rem anyways.

Reply:
Ok that makes sense!!